### ISSUE TITLE
Disabling virtual desktop switching with the mousewheel.

### ISSUE THREAD

**Issue by BobbyBabes** [martenjj/wallpaperswitch#4]
First of all thanks for this utility. It's great to have a different wallpaper on each desktop again.
I switched from Deepin Desktop, which has about the coolest Virtual Desktop Manager.

I tried disabling virtual desktop switching with the mousewheel in `System Settings > Desktop Behavior > Virtual Desktops > Switching-tab`. But there is nothing for it.

Is it possible in wallpaperswitch to disable switching with the mousewheel ?

**Comment by martenjj** [comment]
This isn't handled by the wallpaper switcher, and the setting is not in the obvious place as above.  Right click over the desktop background and go to Configure Desktop - Mouse Actions.  If there is an action here for "Vertical Scroll" assigned to "Switch Desktop", then either remove it with the red "-" or set it to another action.  This should then disable the desktop switching on mouse wheel.

**Comment by BobbyBabes** [comment]
Thanks I fixed it.

It's a feature/bug in KDE.
The first thing I did was what you described. It didn't work. (I know now why, because you pointed me back to it.)
When removing that mouse action didn't work, I went into Desktop Behavior. Nothing there. So I opened this issue.

What I should have done, was recheck Mouse Actions before opening this issue.
Because the Vertical-Scroll Switch Desktop was back.
When I delete this action with the third small square button (with the red icon on it) it disappears.
But the Apply button isn't activated. It remains greyed out. So I close the window. I should have clicked the OK button before. Very awkward.
